contao/library/Contao/File.php

```diff
@@ -681,32 +683,23 @@ public function resizeTo($width, $height, $mode='')
 	 */
 	public function sendToBrowser($filename=null)
 	{
-		// Make sure no output buffer is active
-		// @see http://ch2.php.net/manual/en/function.fpassthru.php#74080
-		while (@ob_end_clean());
-
-		// Prevent session locking (see #2804)
-		session_write_close();
-
-		// Disable zlib.output_compression (see #6717)
-		ini_set('zlib.output_compression', 'Off');
-
-		// Open the "save as …" dialogue
-		header('Content-Type: ' . $this->mime);
-		header('Content-Transfer-Encoding: binary');
-		header('Content-Disposition: attachment; filename="' . ($filename ?: $this->basename) . '"');
-		header('Content-Length: ' . $this->filesize);
-		header('Cache-Control: must-revalidate, post-check=0, pre-check=0');
-		header('Pragma: public');
```

Comment:
Of course I tested! And the headers are either set by default via the BinaryFileResponse class or useless. I was pretty careful about that! It works like a charm for me :)